db/migrate/20160830061749_create_solidus_paypal_braintree_sources.rb

```diff
@@ -1,6 +1,6 @@
 class CreateSolidusPaypalBraintreeSources < SolidusSupport::Migration[4.2]
```

Comment:
I didn't rename the migration filenames and class names because if renamed, `bin/rails railties:install:migrations` would interpret the migrations as new migrations for SolidusBraintree.